docs/installation.rst

```diff
@@ -170,6 +170,8 @@ Many of Pillow's features require external libraries:
   * Windows support: Libimagequant requires VS2013/MSVC 18 to compile,
     so it is unlikely to work with any Python prior to 3.5 on Windows.
 
+* **libraqm** provides complex text layout support.
+
```

Comment:
Need to mention the other libraries. Also, below there are build flags that need to be updated.